src/Treemap.js

```diff
@@ -22,8 +22,13 @@ export default class Treemap extends Viz {
     super();
 
     this._layoutPadding = 1;
+    this._rankIndices = [];
     this._shapeConfig = assign({}, this._shapeConfig, {
-      ariaLabel: (d, i) => `${d.index + 1}. ${this._drawLabel(d, i)}, ${d.value}.`,
+      ariaLabel: (d, i) => {
+        console.log("d: ", d);
+        console.log("rank:", this._rankData ? this._rankData.indexOf(d) + 1 : false);
```

Comment:
you're consoling the new value, but aren't actually using it! please use it, and then delete these 2 console.log statements